src/main/java/org/ambraproject/wombat/service/remote/orcid/OrcidApi.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2017 Public Library of Science
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+package org.ambraproject.wombat.service.remote.orcid;
+
+import org.ambraproject.wombat.config.site.Site;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+/**
+ * Interface to the ORCID API
+ */
+public interface OrcidApi {
+
+  /**
+   * Authenticates a code. If successful, return the associated ORCID ID.
+   *
+   * @return ORCID ID
+   * @throws IOException
+   */
+  public String getOrcidIdFromCode(Site site, String code) throws IOException, URISyntaxException;
+
```

Comment:
Does getOrcidIdFromAuthCode make more sense?

Reply:
Yes it does, thanks!